views/layout.jade

```diff
@@ -0,0 +1,7 @@
+doctype html
```

Comment:
how this file is used?

Reply:
I used npx express-generator so I think there is a fair amount of excess boilerplate. I will get rid of it all.